schemas/common_definitions.json

```diff
@@ -18,6 +18,14 @@
     "description": "Allows customisation of question definition title and description.",
     "type": "array",
     "minItems": 1,
+    "maxItems": 1,
+    "items": {
+      "$ref": "#/definition"
+    }
+  },
+  "question_definition": {
+    "description": "Allows customisation of question definition title and description.",
+    "type": "object",
     "items": {
```

Comment:
"items" is for an array. Is this even needed?. You can just use `definition` directly?